src/logger.py

```diff
@@ -0,0 +1,62 @@
+import aiogram
+import logging
+from src.constants import *
+import types
+
+class NoFapLogger(object):
+    _commandLogger = None
+    _cronLogger = None
+
+    _instance = None
+
+    @staticmethod
+    def _turnOnConsoleLogging(logger):
+        formatter = logging.Formatter("%(asctime)s %(levelname)s %(message)s")
+
+        stream_handler = logging.StreamHandler()
+        stream_handler.setLevel(logging.INFO)
+        stream_handler.setFormatter(formatter)
+
+        logger.addHandler(stream_handler)
+
+    @staticmethod
+    def _turnOffConsoleLogging(logger):
+        # I have no idea how to turn off logging into console.
+        # Approach with remove StreamHandler from logger doesn't working
+        pass

```

Comment:
https://stackoverflow.com/questions/2266646/how-to-disable-logging-on-the-standard-error-stream